COVIDWatch iOS/UIComponents/Header.swift

```diff
@@ -9,9 +9,10 @@
 import UIKit
 
 class Header: UIView {
+    var parentVC: BaseViewController?
```

Comment:
This will leak memory since both classes have a strong reference to each other.  Can you please make this one a weak var?

Reply:
Good catch